src/main/java/frc/robot/Telemetry.java

```diff
@@ -39,17 +37,23 @@ public Telemetry(double maxSpeed) {
     /* Robot swerve drive state */
     private final NetworkTable driveStateTable = inst.getTable("DriveState");
     private final StructPublisher<Pose2d> drivePose = driveStateTable.getStructTopic("Pose", Pose2d.struct).publish();
+    private final StructPublisher<Pose2d> visionPose = driveStateTable.getStructTopic("Vision Pose", Pose2d.struct).publish();
```

Comment:
As now you have a Field2d field this visionPose and the drivePose above both can be deleted.